Phonebook.Frontend/src/app/shared/dialogs/dialogs.module.ts

```diff
@@ -5,11 +5,13 @@ import { DisplayNotificationDialog } from 'src/app/shared/dialogs/display-notifi
 import { ReleaseNotificationDialog } from 'src/app/shared/dialogs/release-notification-dialog/release-notification.dialog';
 import { MaterialModule } from 'src/app/shared/material.module';
 import { ContributorsModule } from 'src/app/shared/components/contributors/contributors.module';
+import { UserInformationComponent } from './user-information/user-information.component';
+import { CommonModule } from '@angular/common';
 
 @NgModule({
-  declarations: [ReleaseNotificationDialog, DisplayNotificationDialog, BugReportConsentComponent],
-  imports: [RouterModule, MaterialModule, ContributorsModule],
-  exports: [ReleaseNotificationDialog, DisplayNotificationDialog, BugReportConsentComponent],
-  entryComponents: [ReleaseNotificationDialog, DisplayNotificationDialog, BugReportConsentComponent]
+  declarations: [ReleaseNotificationDialog, DisplayNotificationDialog, BugReportConsentComponent, UserInformationComponent],
```

Comment:
Please Rename to something more fitting as @mschwrdtnr already said: IncorrectUserInformationComponent